tests/shipping-preview/Delivery Only_Delivery Pickup - Credit card - vtexgame1nolean.test.js

```diff
@@ -0,0 +1,3 @@
+import test from './models/Delivery Only_Delivery Pickup - Credit card.model.js'
+
+test('vtexgame1nolean')
```

Comment:
Why is this the only file that doesn't end with a newline? 🤔

Reply:
Oops! Fixed.